types/jsforce/api/metadata.d.ts

```diff
@@ -1,7 +1,6 @@
 import { Callback, Connection } from '../connection';
 import { EventEmitter } from 'events';
 import { Stream } from 'stream';
-import { Buffer } from 'buffer';
```

Comment:
Removed otherwise got a DefinitelyTyped error about referencing outside libraries. Noticed this import wasn't used to removed it.

```shell
$ npm test jsforce

> definitely-typed@0.0.3 test
> dtslint types "jsforce"

Error: /Users/doug.ayers/Documents/workspace/DefinitelyTyped/node_modules/buffer/index.d.ts:1:1
ERROR: 1:1  no-outside-dependencies  File /Users/doug.ayers/Documents/workspace/DefinitelyTyped/node_modules/buffer/index.d.ts comes from a `node_modules` but is not declared in this type's `package.json`.  See: https://github.com/Microsoft/dtslint/blob/master/docs/no-outside-dependencies.md

    at /Users/doug.ayers/Documents/workspace/DefinitelyTyped/node_modules/dtslint/bin/index.js:206:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/doug.ayers/Documents/workspace/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
npm ERR! code 1
npm ERR! path /Users/doug.ayers/Documents/workspace/DefinitelyTyped
npm ERR! command failed
npm ERR! command sh -c dtslint types "jsforce"
```